src/error.rs

```diff
@@ -37,6 +39,13 @@ impl Display for Error {
             #[cfg(target_os = "linux")]
             Self::Seccomp(error) => write!(f, "seccomp error: {error}"),
             Self::InvalidPath(path) => write!(f, "invalid path: {path:?}"),
+            #[cfg(target_os = "linux")]
+            Self::Io(error) if error.kind() == IoErrorKind::Unsupported => {
+                write!(
+                    f,
+                    "unsupported operation, please ensure Kernel version is at least 5.12: {error}"
```

Comment:
`5.12` appears to be a bit of a magic number. Will it always be this value? If not, can it be abstracted somewhere as a constant and made to be more obvious...in both code updates which might change it and in documentation? The only current documentation that could be found in this repository, regarding minimum supported kernel version, is [in the CHANGELOG](https://github.com/phylum-dev/birdcage/blob/main/CHANGELOG.md?plain=1#L49) where `3.8` is the stated value.

Reply:
This number is purely a guess at what might give people a good chance of getting things to work if people run into this issue.

The idea here is mostly to just give some rough advice and if someone comes back and says it doesn't work with 5.12 either, we can just update the error.

At the end of the day you can always run the latest kernel with some features removed and still have it not work. That's why I think it makes sense to keep this small and contained, I don't really want people to rely on this for anything but troubleshooting when things already don't work.